src/components/pages/Offers/Offers/__specs__/Offers.spec.jsx

```diff
@@ -782,10 +780,8 @@ describe('src | components | pages | Offers | Offers', () => {
     it('should load offers with selected venue filter', async () => {
       // Given
       await renderOffers(props, store)
-      const venueSelect = screen.getByDisplayValue(ALL_VENUES_OPTION.displayName, {
-        selector: 'select[name="lieu"]',
-      })
-      fireEvent.change(venueSelect, { target: { value: proVenues[0].id } })
+      const venueSelect = screen.getByLabelText('Lieu')
+      userEvent.selectOptions(venueSelect, await within(venueSelect).findByText(proVenues[0].name))
```

Comment:
```suggestion
      const firstVenueOption = await within(venueSelect).findByText(proVenues[0].name);
      userEvent.selectOptions(venueSelect, firstVenueOption)
```
c'est un peu plus clair